doc/FAQ.md

```diff
@@ -183,25 +183,48 @@ Example: Friendica Support
 ### What friendica clients can I use?
 
 Friendica is using a [Twitter/GNU Social compatible API](help/api), which means you can use any Twitter/GNU Social client for your platform as long as you can change the API path in its settings.
-Since the 2021.06 release, Friendica also supports the Mastodon API.
+Since the 2021.06 release, Friendica also supports the [Mastodon API](help/API-Mastodon) which works with a lot of Mastodon clients.
```

Comment:
Or `most` if it's relevant.
```suggestion
Since the 2021.06 release, Friendica also supports the [Mastodon API](help/API-Mastodon) which works with many Mastodon clients.
```